Statically link markdown

Hello @matsumoto-r!

I'm trying to statically link discount for your gem, here's my attempt at `mrbgem.rake`:

```ruby
MRuby::Gem::Specification.new('mruby-discount') do |spec|
  spec.license = 'MIT'
  spec.authors = 'MATSUMOTO Ryosuke'
  spec.linker.libraries << 'markdown'
  require 'open3'

  discount_dir = "#{build_dir}/discount"
  markdowna = "#{discount_dir}/libmarkdown.a"

  def run_command env, command
    STDOUT.sync = true
    puts "build: [exec] #{command}"
    Open3.popen2e(env, command) do |stdin, stdout, thread|
      print stdout.read
      fail "#{command} failed" if thread.value != 0
    end
  end

  FileUtils.mkdir_p build_dir

  if ! File.exists? discount_dir
    Dir.chdir(build_dir) do
    e = {}
      run_command e, 'git clone git://github.com/Orc/discount.git'
    end
  end

  if ! File.exists? markdowna
    Dir.chdir discount_dir do
      e = {}
      run_command e, './configure.sh'
      run_command e, 'make'
    end
  end

  spec.linker.libraries << markdowna
end
```

But still, when I try to compile this I get the following:

```
CC    ../mruby-discount/src/mrb_discount.c -> build/host/mrbgems/mruby-discount/src/mrb_discount.o
/Users/zzak/drive/hacks/static/mruby-discount/src/mrb_discount.c:30:10: fatal error: 'mkdio.h' file not found
#include <mkdio.h>
```

Any ideas, I'm welcome to handle linking outside of this gem (if possible).

Thank you!!

--- a/mrbgem.rake
+++ b/mrbgem.rake
@@ -2,4 +2,36 @@ MRuby::Gem::Specification.new('mruby-discount') do |spec|
   spec.license = 'MIT'
   spec.authors = 'MATSUMOTO Ryosuke'
   spec.linker.libraries << 'markdown'
+  require 'open3'
+
+  discount_dir = "#{build_dir}/discount"
+
+  def run_command env, command
+    STDOUT.sync = true
+    puts "build: [exec] #{command}"
+    Open3.popen2e(env, command) do |stdin, stdout, thread|
+      print stdout.read
+      fail "#{command} failed" if thread.value != 0
+    end
+  end
+
+  FileUtils.mkdir_p build_dir
+
+  if ! File.exists? discount_dir
+    Dir.chdir(build_dir) do
+      e = {}
+      run_command e, 'git clone git://github.com/Orc/discount.git'
+    end
+  end
+
+  if ! File.exists? "#{discount_dir}/libmarkdown.a"
+    Dir.chdir discount_dir do
+      e = {}
+      run_command e, './configure.sh'
+      run_command e, 'make'
+    end
+  end
+
+  spec.cc.include_paths << discount_dir
+  spec.linker.library_paths << discount_dir
 end

